src/app/global/utils.nim

```diff
@@ -154,8 +154,11 @@ QtObject:
     except:
       return false
 
+  proc isHexFormat*(self: Utils, value: string): bool {.slot.} =
+    return value.match(re2"^0x[0-9a-fA-F]+$")
+
   proc isChatKey*(self: Utils, value: string): bool {.slot.} =
-      result = (conversion.startsWith0x(value) and conversion.isHexFormat(value) and len(value) == 132) or self.isCompressedPubKey(value)
+    result = (conversion.startsWith0x(value) and self.isHexFormat(value) and len(value) == 132) or self.isCompressedPubKey(value)
```

Comment:
`0x` is checked in `isHexFormat` now, so `conversion.startsWith0x` is redundant, isn't it?

Reply:
(I can feel your hate levels towards me increasing)

Reply:
If only you weren't right each time, then I could hate you freely :cry: